.travis.yml

```diff
@@ -15,5 +15,5 @@ install:
       tar -C $TRAVIS_BUILD_DIR/.. -xf $ECLIPSE_TAR
       export ECLIPSE_EXE=$TRAVIS_BUILD_DIR/../eclipse/eclipse
       export JDT=$TRAVIS_BUILD_DIR/ecj.jar
-      wget http://ftp.halifax.rwth-aachen.de/eclipse/eclipse/downloads/drops4/R-4.5.2-201602121500/ecj-4.5.2.jar -O $JDT
+      wget https://lafo.ssw.uni-linz.ac.at/pub/sulong-deps/ecj-4.5.2.jar -O $JDT
```

Comment:
We could use http://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops4/R-4.5.2-201602121500/ecj-4.5.2.jar instead. Would that be more reliable?